### PR TITLE
PLAT-139507: Fixed Layout bug for inline item on Gallium skin

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed Layout bug for inline item with label and labelPosition "after" or "before" on Gallium skin to not change item width on hover. Also updated screenshot and ui tests.

### Links
[//]: # (Related issues, references)
PLAT-139507

### Comments

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: styles/variables-base.less [hunk 1]
```diff
@@ -415,18 +415,18 @@
 
 // Item
 // ---------------------------------------
-@agate-item-border-width:             inherit;
+@agate-item-border-width:             0;
 @agate-item-content-width:            60%;
-@agate-item-centered-inline-max-width: @agate-item-inline-max-width;
+@agate-item-centered-inline-max-width: @agate-item-inline-min-width;
```
Comment (jeonghee27) [inline styles/variables-base.less:420]: Why we should change this?

**Review by jeonghee27 — APPROVED** [review]
LGTM. Please check @agate-item-centered-inline-max-width when merge

File: styles/variables-base.less (hunk 1 above)
Comment (stanca-pop-lgp) [reply styles/variables-base.less:420]: Yes, also check "centered" knob. It is only valid for centered. If I undo this change, when hovered, the item becomes wider. 
For inline item with centered=false the max-width is indeed @agate-item-inline-max-width.

**Pull request merged**